packages/manager/apps/ips/src/utils/getCatalog.ts

```diff
@@ -0,0 +1,136 @@
+import { useQuery } from '@tanstack/react-query';
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { useContext } from 'react';
+
+export const BYOIP_FAILOVER_V4 = 'byoip-failover-v4';
+export const CONFIG_NAME = {
+  CAMPUS: 'campus',
+  IPRIR: 'ipRir',
+} as const;
+
+export type Campus = {
+  name: string;
+  planCode: string;
+};
+
+export type ProductConfiguration = {
+  name: string;
+  values: string[] | Campus[];
+};
+
+export type Product = {
+  description: string;
+  internalType: string;
+  name: string;
+  configurations: ProductConfiguration[];
+};
+
+export type PlanDetails = {
+  metadatas: Record<string, unknown>;
+  pricings: Record<string, unknown>;
+  product: Product;
+};
+
+export type Plan = {
+  addonsFamily: string;
+  consumptionBillingStrategy: string;
+  details: PlanDetails;
+  invoiceName: string;
+  pricingType: string;
+  planCode: string;
+};
+
+export type CatalogResponse = {
+  plans: Plan[];
+};
+
+export const useCatalog = () => {
+  const { shell } = useContext(ShellContext);
```

Comment:
`getCatalog.ts` should be in the `data` folder, not the `utils` folder
In api and hooks folders